source/using-web-interface.rst

```diff
@@ -277,6 +280,14 @@ This section will demonstrate how to generate a device history report. The repor
     src="https://www.youtube.com/embed/G1ZFfCLUT6A?rel=0" frameborder="0"
     allowfullscreen></iframe>
 
+Performing access to the Dashboard
+------------------
+
+The dashboard is part of gui-v2, to access it you need to use another url, in the current version just add to the end of the url “/v2” in relation to the interface used in the previous items, for example, in the case of localhost it would be http://localhost:8000/v2, see more at :doc:`components-and-apis`.
+The login and password credentials are the same as those used in the rest of Dojot. After logging in, a new screen will open and an ADD button will appear in the upper right corner, which will give you the options for various types of viewing. At this moment, the parameters will be configured, in the first “General” screen it will be necessary to add a name for the visualization and optionally a description.
```

Comment:
> [...] will open and an `ADD` button will appear [...] 

You can format menu and button names as if they were `code`. The same goes for IPs and localhost.